src/components/CreateGameForm.tsx

```diff
@@ -52,31 +58,31 @@ const Button = styled.div`
 const CreateGameForm = () => {
   const [isOpen, setIsOpen] = useState<boolean>(false);
   const [input, setInput] = useState<string>("");
+  const roomNameRegex = /^\w+$/;
+  const isValid = roomNameRegex.test(input);
+  const textInput = useRef<WiredInput>(null);
```

Comment:
you don't need useRef if you use onInput={} in wired-element. It still works, at least for me.